spikes/npgsql-vanilla/README.rst

```diff
@@ -0,0 +1,40 @@
+#################################################################
+Basic demonstration program for using CrateDB with vanilla Npgsql
+#################################################################
+
+
+************
+Introduction
+************
+
+CrateDB versions prior to 4.2 needed a custom fork of the official Npgsql .NET
+data provider for PostgreSQL, `crate-npgsql`_. CrateDB versions 4.2 and later
+work with the vanilla Npgsql driver without the need for a plugin.
+
+The file ``DemoProgram.cs`` is a demonstration program written in C# which
+outlines typical, very basic usage scenarios with CrateDB.
+ 
+
+*****
+Usage
+*****
+
+In order to invoke a CrateDB instance for evaluation purposes, run::
+
+    docker run -it --rm --publish=4200:4200 --publish=5432:5432 crate:4.5.1
+
+For building and running the demonstration program and connect to
+``localhost:5432``, invoke::
+
+    dotnet run 
+
+In order to connect to CrateDB Cloud, use::
+
+    dotnet run -- --host=clustername.aks1.westeurope.azure.cratedb.net --ssl-mode=Require --username=foobar --password='X8F%Shn)TESTvF5ac7%eW4NM'
```

Comment:
Probably doesn't do any harm, but how about replacing the password with a placeholder as well?

Reply:
I thought it would be a good idea to also demonstrate how a regular CrateDB Cloud password looks like. If you think this is not appropriate, I can adjust this.